packages/modal/CHANGELOG.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "pcln-modal",
   "entries": [
+    {
+      "version": "4.2.0",
+      "tag": "pcln-modal_v4.2.0",
+      "date": "Wed, 16 Mar 2022 19:07:49 GMT",
+      "comments": {
+        "minor": [
+          {
+            "comment": "added textStyle prop to ModalHeader and color prop to SmallModalHeader"
+          }
+        ]
+      }
+    },
```

Comment:
should this have stayed a rush change file?

Reply:
What does this mean?

Reply:
you can submit the generated file and then the CI will use it to generate these files but you did that step manually, I don't think it's an issue though.

Reply:
ok cool, updated